packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/member-ordering */
```

Comment:
nit, feel free to leave as-is, but it doesn't look like it'd be too hard to fix this suppression

Reply:
I had a couple more things to clean-up there I was keeping in my back pocket but went ahead and spelled it out in an issue for follow-up https://github.com/ni/nimble/issues/2178